packages/shopping/src/controllers/user-management.controller.ts

```diff
@@ -97,6 +105,7 @@ export class UserManagementController {
     validateCredentials(_.pick(newUserRequest, ['email', 'password']));
 
     try {
+      newUserRequest.resetKey = '';
```

Comment:
I'm wondering if we should the more semantically-correct database NULL instead of a blank string.

Reply:
I think this is fine for a NoSQL database. If we were using SQL database it would make more sense to